src/desktop/apps/artist/client/index.coffee

```diff
@@ -5,8 +5,15 @@ Artist = require '../../../models/artist.coffee'
 CurrentUser = require '../../../models/current_user.coffee'
 ArtistRouter = require './router.coffee'
 FollowedArtistsRailView = require '../../home/components/followed_artists/view.coffee'
+splitTest = require '../../../components/split_test/index.coffee'
+sd = require('sharify').data
```

Comment:
this may already be `require` here above, in line 1 of this file

Reply:
:+1: I have removed this duplication